### PR TITLE
Closes #66 Enable overriding leftmeta with the ansible playbook

### LINKED ISSUE #66: capslock
I noticed you have set the meta key to be capslock (https://github.com/jmontleon/pixelbook-fedora/commit/2ceae9ce376b028b312ab5a07fc3b52309e10399) -- however, I do prefer the old, out-of-the-box option of having that key as the meta key. Is there a way I can switch it back?

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
@huang-jy does this look reasonable to you? I tried it back and forth and it seems to do the right thing...

File: ansible/playbook.yml [hunk 1]
```diff
@@ -270,6 +271,40 @@
...
       - pixelbook-keyboard-backlight
 
+    - name: Ensure keyboard hwdb is not overridden
+      become: true
+      file:
+        path: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+        state: absent
+      when: leftmeta == "capslock"
+
+    - name: Create keyboard hwdb override
```
Comment (huang-jy) [inline ansible/playbook.yml:281]: L281 -> L294 only runs if user sets the `leftmeta` var to `"leftmeta"` . If the user wanted to bind something _else_ to the `leftmeta` key, these steps would not run.

I think using `when: leftmeta != "capslock"` (or similar) might be better here.

**Comment by huang-jy** [comment]
Yes, just one minor suggestion as above

File: ansible/playbook.yml (hunk 1 above)
Comment (jmontleon) [reply ansible/playbook.yml:281]: Sure. Seems reasonable!

File: ansible/playbook.yml [hunk 2]
```diff
@@ -270,6 +271,40 @@
...
+        src: /usr/lib/udev/hwdb.d/61-eve-keyboard.hwdb
+        dest: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+      when: leftmeta != "capslock"
+
+    - name: Set leftmeta to leftmeta in hwdb override
+      become: true
+      lineinfile:
+        path: /etc/udev/hwdb.d/61-eve-keyboard.hwdb
+        regexp: '^ KEYBOARD_KEY_db=capslock$'
+        line: ' KEYBOARD_KEY_db=leftmeta'
```
Comment (huang-jy) [inline ansible/playbook.yml:293]: Does the `leftmeta` in L293 evaluate to the string `"leftmeta"` or the _variable_ `leftmeta` if the former, then regardless of what the user specifies as override, if it's not `capslock` it'll set the override `leftmeta`.

**Pull request merged**